qutebrowser/javascript/webelem.js

```diff
@@ -188,26 +278,56 @@ window._qutebrowser.webelem = (function() {
             return null;
         }
 
-        return serialize_elem(elem);
+        // Check if we got an iframe, and if so, recurse inside of it
+        const frame_elem = replace_elem_frame(elem,
+            (frame) => serialize_elem(frame.document.activeElement, frame));
+
+        if (frame_elem === null) {
+            return serialize_elem(elem);
```

Comment:
That check seems unnecessary as `serialize_elem` does handle `null`.

Reply:
I think this logic is slightly different than that, Here I'm checking if we failed to find an iframe above, and if so, serialize the element and return it. Otherwise, I want to return the frame element. If I just return `serialize_elem(elem)` it dosen't quite get the logic I want